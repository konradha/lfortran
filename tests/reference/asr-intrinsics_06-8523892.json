{
    "basename": "asr-intrinsics_06-8523892",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/intrinsics_06.f90",
    "infile_hash": "45568686a8770f0612e168afdd5bfba81ef25b7ad61daf3ef5960d15",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-intrinsics_06-8523892.stderr",
    "stderr_hash": "d894427f046ca8bb7d9e101e3f0ba0029398d843be7345206c786e93",
    "returncode": 2
}