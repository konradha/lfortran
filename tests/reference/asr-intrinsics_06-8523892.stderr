semantic error: Function 'datan' not found or not implemented yet (if it is intrinsic)
  --> tests/../integration_tests/intrinsics_06.f90:13:5
   |
13 | x = datan(1.5574077)
   |     ^^^^^^^^^^^^^^^^ 
